src/molecules/Divider/Divider.tsx

```diff
@@ -4,11 +4,17 @@ import './Divider.scss';
 export interface Props {
   text: string;
   type?: 'no-line' | 'two-line';
+  verticalPadding?: string;
 }
 
-export const Divider: React.FC<Props> = ({ text, type }) => {
+export const Divider: React.FC<Props> = ({ text, type, verticalPadding }) => {
+  let padding = '';
+  if (verticalPadding) {
+    padding = `${verticalPadding} 0`;
+  }
+
   return (
-    <div className={'Divider ' + (type ? `Divider--${type}` : '')}>
+    <div className={'Divider ' + (type ? `Divider--${type}` : '')} style={{ padding }}>
```

Comment:
Don't hardcode this — rather just extend the div html type then spread the rest props on this. That way you can just natively handle this at the callsite.

Reply:
`{...rest}`